### ISSUE TITLE
WindowsError: [Error 2] - on file finder

### ISSUE THREAD

**Issue by jaraco** [pypa/setuptools-scm#27]
Windows 7, Python 2.7.8

To reproduce:

```
git clone https://github.com/peritus/bumpversion bumpversion
cd bumpversion
python.exe setup.py sdist
```

```
C:\Users\<user>\Documents\scripts\python\_other\bumpversion_clean2>python.exe setup.py sdist
C:\Python27\ArcGIS10.2\lib\site-packages\setuptools\dist.py:282: UserWarning: Normalizing '0.5.4-dev' to '0.5.4.dev0'
  normalized_version,
options (after parsing config files):
options (after parsing command line):
option dict for 'aliases' command:
  {}
option dict for 'sdist' command:
  {}
option dict for 'wheel' command:
  {'universal': ('setup.cfg', '1')}
running sdist
Distribution.get_command_obj(): creating 'sdist' command object
running egg_info
Distribution.get_command_obj(): creating 'egg_info' command object
writing bumpversion.egg-info\PKG-INFO
writing top-level names to bumpversion.egg-info\top_level.txt
writing dependency_links to bumpversion.egg-info\dependency_links.txt
writing entry points to bumpversion.egg-info\entry_points.txt
Distribution.get_command_obj(): creating 'build_py' command object
Distribution.get_command_obj(): creating 'build' command object
error: [Error 2]
Traceback (most recent call last):
  File "setup.py", line 39, in <module>
    'Programming Language :: Python :: Implementation :: PyPy',
  File "C:\Python27\ArcGIS10.2\lib\distutils\core.py", line 151, in setup
    dist.run_commands()
  File "C:\Python27\ArcGIS10.2\lib\distutils\dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "C:\Python27\ArcGIS10.2\lib\distutils\dist.py", line 972, in run_command
    cmd_obj.run()
  File "C:\Python27\ArcGIS10.2\lib\site-packages\setuptools\command\sdist.py", line 40, in run
    self.run_command('egg_info')
  File "C:\Python27\ArcGIS10.2\lib\distutils\cmd.py", line 326, in run_command
    self.distribution.run_command(command)
  File "C:\Python27\ArcGIS10.2\lib\distutils\dist.py", line 972, in run_command
    cmd_obj.run()
  File "C:\Python27\ArcGIS10.2\lib\site-packages\setuptools\command\egg_info.py", line 180, in run
    self.find_sources()
  File "C:\Python27\ArcGIS10.2\lib\site-packages\setuptools\command\egg_info.py", line 207, in find_sources
    mm.run()
  File "C:\Python27\ArcGIS10.2\lib\site-packages\setuptools\command\egg_info.py", line 291, in run
    self.add_defaults()
  File "C:\Python27\ArcGIS10.2\lib\site-packages\setuptools\command\egg_info.py", line 323, in add_defaults
    rcfiles = list(walk_revctrl())
  File "C:\Python27\ArcGIS10.2\lib\site-packages\setuptools\command\sdist.py", line 19, in walk_revctrl
    for item in ep.load()(dirname):
  File "C:\Python27\ArcGIS10.2\lib\site-packages\setuptools_scm\__init__.py", line 77, in find_files
    return do(ep.load(), path).splitlines()
  File "C:\Python27\ArcGIS10.2\lib\site-packages\setuptools_scm\utils.py", line 48, in do
    out, err, ret = do_ex(cmd, cwd)
  File "C:\Python27\ArcGIS10.2\lib\site-packages\setuptools_scm\utils.py", line 33, in do_ex
    HGPLAIN='1',
  File "C:\Python27\ArcGIS10.2\lib\subprocess.py", line 710, in __init__
    errread, errwrite)
  File "C:\Python27\ArcGIS10.2\lib\subprocess.py", line 958, in _execute_child
    startupinfo)
WindowsError: [Error 2]
```

Contents of setup.py

```
import re
from setuptools import setup

description = 'Version-bump your software with a single command!'

long_description = re.sub(
  "\`(.*)\<#.*\>\`\_",
  r"\1",
  str(open('README.rst', 'rb').read()).replace(description, '')
)

setup(
    name='bumpversion',
    version='0.5.4-dev',
    url='https://github.com/peritus/bumpversion',
    author='Filip Noetzel',
    author_email='filip+bumpversion@j03.de',
    license='MIT',
    packages=['bumpversion'],
    description=description,
    long_description=long_description,
    entry_points={
        'console_scripts': [
            'bumpversion = bumpversion:main',
        ]
    },
    classifiers=(
        'Development Status :: 4 - Beta',
        'Intended Audience :: Developers',
        'License :: OSI Approved :: MIT License',
        'Operating System :: OS Independent',
        'Programming Language :: Python',
        'Programming Language :: Python :: 2',
        'Programming Language :: Python :: 2.7',
        'Programming Language :: Python :: 3',
        'Programming Language :: Python :: 3.2',
        'Programming Language :: Python :: 3.3',
        'Programming Language :: Python :: 3.4',
        'Programming Language :: Python :: Implementation :: PyPy',
    ),
)
```

Error 2 means "file not found"
https://msdn.microsoft.com/en-us/library/windows/desktop/ms681382%28v=vs.85%29.aspx

But which file? There is no information.

When i delete setuptools_scm `pip uninstall setuptools_scm` - all is ok, no more error.

---
- Bitbucket: https://bitbucket.org/pypa/setuptools_scm/issue/27
- Originally reported by: [lansman](http://bitbucket.org/lansman)
- Originally created at: 2015-05-20T11:48:19.794

**Comment by jaraco** [comment]
File finders again

---

Original comment by: Ronny Pfannschmidt

**Comment by jaraco** [comment]
print exception on failed/broken file finding, fixes #27

→ <<cset fa3dd4d12a4a>>

---

Original comment by: Ronny Pfannschmidt
